graphql_compiler/schema/sqlalchemy_schema.py

```diff
@@ -1,17 +1,111 @@
 # Copyright 2019-present Kensho Technologies, LLC.
 from collections import namedtuple
 
+from graphql.type.definition import GraphQLInterfaceType, GraphQLObjectType
+import six
+import sqlalchemy
+
+from . import is_vertex_field_name
+
+
+# Describes the intent to join two tables on equality of a column of theirs.
+# The resulting join expression could be something like:
+# JOIN origin_table.from_column = destination_table.to_column
+#
+# The type of join (inner vs left, etc.) is not specified.
+# The tables are not specified.
+DirectJoinDescriptor = namedtuple('DirectJoinDescriptor', (
+    'from_column',  # The column in the source table we intend to join on.
+    'to_column',    # The column in the destination table we intend to join on.
+))
+
+
+# TODO(bojanserafimov): Define a JunctionJoinDescriptor
```

Comment:
Can you briefly mention what is a JunctionJoinDescriptor and why need it if we already have a DirectJoinDescriptor?

Reply:
Removed the TODO. When you have a need for a `JunctionJoinDescriptor` you'll figure out where to put it.